bin/logify.pl

```diff
@@ -19,29 +21,59 @@
 	die "$ARGV:$.: error: missing closing parenthesis$/"
 };
 
+my $script = $FindBin::Script;
+my $usage = <<"EOF";
+Usage: $script [options] filename ...
+Options:
+  [-e|--exclude]	Comma-separated list of methods to exclude
+	 -e "launchedTaskWithLaunchPath:arguments:,arguments" (for example)
+  [-f|--filter]		Comma-separated list of methods to exclude
```

Comment:
It might be better to use "include" as contrast to "exclude" (and fix the help text)

Reply:
Good call!